gateway/gateway.go

```diff
@@ -268,7 +268,14 @@ func (gw *Gateway) handleMessage(msg config.Message, dest *bridge.Bridge) []*BrM
 
 	// only relay topic change when configured
 	if msg.Event == config.EventTopicChange && !gw.Bridges[dest.Account].GetBool("ShowTopicChange") {
-		return brMsgIDs
+		// don't relay topic/purpose change if disabled
+		if !gw.Bridges[dest.Account].GetBool("ShowTopicChange") {
+			return brMsgIDs
+		}
+		// don't relay topic/purpose update if sync is enabled, but msg marked as nosync
+		if gw.Bridges[dest.Account].GetBool("SyncTopic") && strings.HasSuffix(msg.Text, "[nosync]") {
```

Comment:
Unless I misunderstood we are not integrating the `[nosync]` functionality? cc @42wim.

Reply:
wtf. You're 100% right. I think this somehow made it back in while I was rebasing out some other changes. omg this PR is the worst. Ugh. I'm so sorry guys!